java/ql/src/semmle/code/java/dataflow/internal/TaintTrackingUtil.qll

```diff
@@ -341,7 +341,7 @@ private predicate taintPreservingQualifierToMethod(Method m) {
   m.getDeclaringType() instanceof TypeFile and
   m.hasName("toURI")
   or
-  m.getDeclaringType().hasQualifiedName("java.net", "URI") and
+  m.getDeclaringType() instanceof TypeUri and
```

Comment:
Looks like these don't resolve currently. We'll need to add `private import semmle.code.java.Networking`.